gnome/gtk3-devel/Portfile

```diff
@@ -90,7 +90,9 @@ patchfiles-append   patch-quartz-nseventtype.diff
 patchfiles-append   patch-quartz-meson-appleframeworks.diff
 
 # implicit declaration of function 'sincos'
-patchfiles-append   patch-tests-gdkgears.diff
+# https://github.com/mesonbuild/meson/issues/7652
+patchfiles-append   patch-tests-gdkgears.diff \
+                    patch-sincos.diff
```

Comment:
Can you put your new patch on a new line with `patchfiles-append`? This makes it easy to quickly disable a single patch. Otherwise it's necessary to remove the line continuation from the previous line too. (That's why earlier ones are on individual lines.)

Reply:
Yes, sure, will do now.